app/controllers/exercise_exclusions_controller.rb

```diff
@@ -35,7 +36,12 @@ def _update_course_request_payload_schema
         },
         'updated_at': {'$ref': '#/standard_definitions/datetime'}
       },
-      'required': ['request_uuid', 'course_uuid', 'sequence_number', 'exclusions', 'updated_at'],
+      'required': ['request_uuid',
```

Comment:
@nathanstitt was this the correct thing to do?  for 4 service calls, updated the properties and required to require a ecosystem_uuid?